.github/workflows/awsfulltest.yml

```diff
@@ -0,0 +1,69 @@
+name: nf-core AWS full size tests
```

Comment:
this file is only in template for nf-core pipelines

Reply:
same for the other aws test file

Reply:
Done